pkg/metrics/collector/fs.go

```diff
@@ -7,12 +7,67 @@
 package collector
 
 import (
+	"time"
+
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/nydus-snapshotter/pkg/daemon/types"
 	"github.com/containerd/nydus-snapshotter/pkg/metrics/data"
 	mtypes "github.com/containerd/nydus-snapshotter/pkg/metrics/types"
 )
 
+var OPCodeMap = map[uint32]string{
+	1:   "OP_LOOKUP",
+	2:   "OP_FORGET",
+	3:   "OP_GETATTR",
+	4:   "OP_SETATTR",
+	5:   "OP_READLINK",
+	6:   "OP_SYMLINK",
+	8:   "OP_MKNOD",
+	9:   "OP_MKDIR",
+	10:  "OP_UNLINK",
+	11:  "OP_RMDIR",
+	12:  "OP_RENAME",
+	13:  "OP_LINK",
+	14:  "OP_OPEN",
+	15:  "OP_READ",
```

Comment:
Is it necessary to copy all FUSE op code definitions here? Only counting on READ should meet requirements.

Reply:
Yes. If only `READ` operation is available. The `Label` is unnecessary.